pkg/ccl/importccl/import_stmt_test.go

```diff
@@ -5923,7 +5926,7 @@ func TestImportPgDumpDropTable(t *testing.T) {
 		data = `DROP TABLE u;
 		CREATE TABLE u (a INT);
 		INSERT INTO u VALUES (55);
-		DROP TABLE t;`
+		CREATE TABLE t(x int);`
```

Comment:
why has this changed from drop to create?

Reply:
Fixed this.